tools/cert_create/Makefile

```diff
@@ -61,8 +54,7 @@ all: clean ${BINARY}
 
 ${BINARY}: ${OBJECTS} Makefile
 	@echo "  LD      $@"
-	@echo 'const char build_msg[] = "Built : "__TIME__", "__DATE__; \
-                const char platform_msg[] = "${PLAT}";' | \
```

Comment:
Given the current situation of TBB, is it worthwhile?

As I pointed out in the issue "Question: How much memory is required to enable Trusted Board Boot? #448",  TBBR is too over-implemented to fit in typical SRAM size in competetive mass products.  So, TBBR is designed to work on a $1000 board like Juno or emulators.

If you really believe this is the right thing to do, yes, we could do that by sprinkling #ifdef.
But, what does it buy us? and how long should we carry USE_TBBR_DEFS?

One more thing, even if somebody uses their own platform_oid.h in downstream code, this commit is not a breakage.

If the downstream users follow TBBR, it will work seamlessly.
Only the difference is the OIDs will be replaced with ones assigned for ARM Ltd.
This is not a problem because both TF and cert_create reference tbbr_oid.h
instead of platform_oid.h   The users may not even notice that.

If they do not follow TBBR, they are not using drivers/auth/tbbr/tbbr_cot.c.
Instead, they are using their internal authentication table where
platform_oid.h is not mandatory.   They are probably not using cert_create either
because this tool is so hard-coded.

In either way, I could not get the point of you suggestion.